Задание для кандидатов. Выполнить ревью.sql

```diff
@@ -0,0 +1,123 @@
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int
+AS
+set nocount on
+begin
+	declare @RowCount int = (select count(*) from syn.SA_CustomerSeasonal)
+	declare @ErrorMessage varchar(max)
+
+-- Проверка на корректность загрузки
+	if not exists (
+	select 1
+	from syn.ImportFile as f
+	where f.ID = @ID_Record
+		and f.FlagLoaded = cast(1 as bit)
+	)
+		begin
+			set @ErrorMessage = 'Ошибка при загрузке файла, проверьте корректность данных'
+
+			raiserror(@ErrorMessage, 3, 1)
+			return
+		end

```

Comment:
Комментарий 5
Begin и end на уровне if
```suggestion
	and f.FlagLoaded = cast(1 as bit)
	)
	begin
		set @ErrorMessage = 'Ошибка при загрузке файла, проверьте корректность данных'
                raiserror(@ErrorMessage, 3, 1)
		return
	end

```